lib/validators/movie.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).required(),
+  release_year: Joi.number().min(1878).max(9999).optional()
```

Comment:
This should also require that the number be an `integer`, since that is the [schema](https://github.com/pallavi-s/sfmovies/blob/2ae40fc7f7c3a4012ace8a606eb3462af2d976f0/db/migrations/20160608152858_create_movies.js#L7) of the table. As is, this validator would allow `1999.5` to be inserted, but the database would not.